ocs_ci/deployment/deployment.py

```diff
@@ -400,8 +400,9 @@ def deploy_ocs_via_operator(self):
         else:
             logger.info("Deployment of OCS via OCS operator")
             self.label_and_taint_nodes()
-        logger.info("Creating namespace and operator group.")
-        run_cmd(f"oc create -f {constants.OLM_YAML}")
+        if float(config.ENV_DATA['ocs_version']) >= 4.5:
```

Comment:
```suggestion
        if float(config.ENV_DATA['ocs_version']) < 4.5:
```
If I understand correctly, we don't need "deploy-with-olm.yaml" for version less than OCS 4.5 , is that correct ?

Reply:
Yep! you are right!